tickets/src/__tests__/middlewares/walletMiddleware.test.js

```diff
@@ -380,15 +380,16 @@ describe('Wallet middleware', () => {
       signedAddress: 'encrypted sig',
     }
 
-    mockWalletService.recoverAccountFromSignedData = jest.fn(
-      data => data.message.address.publicKey
+    mockWalletService.recoverAccountFromSignedData = jest.fn((data, sa, cb) =>
```

Comment:
this is a good test. We also need a separate test that verifies that the real `walletService.recoverAccountFromSignedData` actually does what we expect it, in order to catch API regressions. Otherwise, should a change to `unlock-js` break this, we won't know until it's deployed to staging. The test would need to mock out the json-rpc call using `nock`, I can definitely help with this. We would borrow `nockHelper` from `unlock-js`

The test would also verify that our assumption that `recoverAccountFromSignedData` calls a callback is correct